index.html

```diff
@@ -4189,6 +4184,10 @@ <h2>
       <h3>Substantive changes since the last published Recommendation</h3>
 
       <ol reversed>
+        <li>
```

Comment:
Note... if you want a lazy way of handling this, you can use [`<rs-changelog>`](https://respec.org/docs/#rs-changelog)... it automatically generates a change log. 

Here is how we use it in Web Manifest:
https://github.com/w3c/manifest/blob/8d8160394073dac4ddf2550aebf90a9ce2043110/index.html#L3223-L3237

And here is what is looks like:
https://www.w3.org/TR/appmanifest/#change-log

Reply:
Thanks. I’ll give that a spin. Doesn’t do the date the change landed though?

Reply:
No, but I’ll see if I can get that info from GitHub.